ballerina/client.bal

```diff
@@ -1,15 +1,993 @@
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->put(resourcePath, request, headers);
+    }
+
+    # Set Repeat Mode
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Command sent 
+    resource isolated function put me/player/repeat(map<string|string[]> headers = {}, *SetRepeatModeOnUsersPlaybackQueries queries) returns http:Response|error {
+        string resourcePath = string `/me/player/repeat`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        http:Request request = new;
+        return self.clientEp->put(resourcePath, request, headers);
+    }
+
+    # Seek To Position
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Command sent 
+    resource isolated function put me/player/seek(map<string|string[]> headers = {}, *SeekToPositionInCurrentlyPlayingTrackQueries queries) returns http:Response|error {
+        string resourcePath = string `/me/player/seek`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        http:Request request = new;
+        return self.clientEp->put(resourcePath, request, headers);
+    }
+
+    # Toggle Playback Shuffle
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Command sent 
+    resource isolated function put me/player/shuffle(map<string|string[]> headers = {}, *ToggleShuffleForUsersPlaybackQueries queries) returns http:Response|error {
+        string resourcePath = string `/me/player/shuffle`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        http:Request request = new;
+        return self.clientEp->put(resourcePath, request, headers);
+    }
+
+    # Set Playback Volume
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Command sent 
+    resource isolated function put me/player/volume(map<string|string[]> headers = {}, *SetVolumeForUsersPlaybackQueries queries) returns http:Response|error {
+        string resourcePath = string `/me/player/volume`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        http:Request request = new;
+        return self.clientEp->put(resourcePath, request, headers);
+    }
+
+    # Save Shows for Current User
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - Show saved 
+    resource isolated function put me/shows(map<string|string[]> headers = {}, *SaveShowsUserQueries queries) returns http:Response|error {
```

Comment:
it seems some APIs are returning `http:Response` instead of proper record types. @lnash94 @TharmiganK need to verify if this is the expected behaviour or, a potential issue in the OpenAPI client generator.
```suggestion
    resource isolated function put me/shows(map<string|string[]> headers = {}, *SaveShowsUserQueries queries) returns http:Response|error {
```

Reply:
In this case, the given operation has a success status code without a specific body, and in the HTTP client, we cannot directly bind the status code as `http:Ok`. The `http:Response` is used because the operation has multiple responses.

If the user is interested in binding the status code, they need to add the `—status-code-binding` flag when generating the code.

Since currently we only consider the success payload, We can check the feasibility of improving the tool to generate `error?` for multiple responses. In the meantime, until we provide this improvement, we can modify the OAS by removing specific error responses.

[1] OAS sample which success response has no content type; Here success response is 200 and it only has a description. Remaining of the status codes are error status code. 
```yml
put:
      tags:
        - Shows
        - Library
      operationId: save-shows-user
      summary: |
        Save Shows for Current User
      description: |
        Save one or more shows to current Spotify user's library.
      parameters:
        - $ref: '#/components/parameters/QueryShowIds'
      responses:
        '200':
          description: Show saved
        '401':
          $ref: '#/components/responses/Unauthorized'
        '403':
          $ref: '#/components/responses/Forbidden'
        '429':
          $ref: '#/components/responses/TooManyRequests'
      security:
        - oauth_2_0:
            - user-library-modify
```
[2] Modified OAS: Simply removed the remaining status code 401, 403, 429 since we bind and process data with the success payloads.

```yml
put:
      tags:
        - Shows
        - Library
      operationId: save-shows-user
      summary: |
        Save Shows for Current User
      description: |
        Save one or more shows to current Spotify user's library.
      parameters:
        - $ref: '#/components/parameters/QueryShowIds'
      responses:
        '200':
          description: Show saved
      security:
        - oauth_2_0:
            - user-library-modify
```